mongodb-centos7-atomicapp/README.md

```diff
@@ -1,9 +1,62 @@
-This is an atomic application based on the nulecule specification. Kubernetes and native docker are currently the only supported providers. You'll need to run this from a workstation that has the atomic command and kubectl client that can connect to a kubernetes master.
+# MongoDB CentOS 7 Atomic App
 
-It's a single container application based on the ``centos/mongodb-26-centos7`` image.
+### Description
 
-### Option 1: Unattended
+A single container application built on the centos/mongodb-26-centos7 image.
 
-Run the image. It will automatically use kubernetes as the orchestration provider.
+### Deployment
+
+#### Deploying with Default Parameters
+
+##### Using Atomic CLI
+
+```sh
+sudo atomic run projectatomic/mongodb-centos7-atomicapp
+```
+
+##### Using Atomic App
+
+```sh
+sudo atomicapp run projectatomic/mongodb-centos7-atomicapp
+```
+
+#### Deploying with User-Provided Parameters
+
+##### Using Atomic CLI
+
+```sh
+sudo atomic run projectatomic/mongodb-centos7-atomicapp --mode fetch --destination mongodb-centos7-atomicapp
+cd mongodb-centos7-atomicapp
+cp answers.conf.sample answers.conf # Modify then copy answers.conf.sample
+sudo atomic run projectatomic/mongodb-centos7-atomicapp .
+```
+
+##### Using Atomic App
+
+```sh
+atomicapp fetch projectatomic/mongodb-centos7-atomicapp --destination mongodb-centos7-atomicapp
+cd mongodb-centos7-atomicapp
+cp answers.conf.sample answers.conf # Modify then copy answers.conf.sample
+atomicapp run .
+```
+
+### Interaction
+
+The default port is :27017
+
+#### Using the Docker Provider
+
+```sh
+telnet localhost 27017
+Trying ::1...
+Connected to localhost.
+Escape character is '^]'.
+```
```

Comment:
would be cool if we had something similar for mongo as we do for mariadb. I made this cheatsheet a while back for myself.. maybe we could include it here:

```
# Connect                                                                                                                                                                                                                                    
mongo -u fum -p foe --host localhost --port 27017 fie                                                                                                                                                                                        
                                                                                                                                                                                                                                      
# Create new collection called "products" and insert new item                                                                                                                                                                                
db.products.insert( { item: "card", qty: 15 } )                                                                                                                                                                                              
                                                                                                                                                                                                                                             
# Print out items from the products collection.                                                                                                                                                                                              
db.products.find()
```

if we just modified this to use a container instead, like the mariadb examples that we have then I think it would be good to have.